Successfully updated:
Created Frame node AWhyyv4Vb
Created Frame node OErY3pRoD
Created Text node VwuxPsBi6
Created Text node hHdErnezz
Created Frame node WDR86ELlt
Updated attributes for node AWhyyv4Vb
Updated attributes for node OErY3pRoD
Updated text for node VwuxPsBi6
Updated attributes for node VwuxPsBi6
Updated text for node hHdErnezz
Updated attributes for node hHdErnezz
Updated attributes for node WDR86ELlt
Reordered node OErY3pRoD within parent AWhyyv4Vb to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -2977,20 +2977,73 @@
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   Body content goes here
 </BodyContentGoesHere>
 <Something
     nodeId="so_W61aYl"
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   something
 </Something>
+<Stack
+    nodeId="AWhyyv4Vb"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="OErY3pRoD"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="VwuxPsBi6"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="hHdErnezz"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="WDR86ELlt"
+      width="100%"
+      height="100px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.